profile.php

```diff
@@ -152,12 +153,13 @@
 					foreach ($product_array as $key => $value) {
 				?>
 						<div class="product-item">
-							<form method="post" action="?action=add&product_id=<?php echo $product_array[$key]["product_id"]; ?>">
+							<form method="post" action="profile.php?action=add&product_id=<?php echo $product_array[$key]["product_id"]; ?>">
```

Comment:
Unnecessary... profile.php add na korleo kaj kore and na use korle borong better... cause taile profile.php er file name change korle or directory change korleo kaj korbe